pytx/pytx/common.py

```diff
@@ -429,6 +433,15 @@ def connections(cls_or_self,
             params[t.FIELDS] = ','.join(f.strip() for f in fields)
         if metadata:
             params[t.METADATA] = 1
+        if summary and fields is not None:
+            params[t.FIELDS] = 'summary'
+        if summary:
+            params[t.FIELDS] = params[t.FIELDS] + ',summary'
```

Comment:
This conditional will always overwrite the one above it. Should the one above it be `if summary and fields is None` and turn this one into an `elif summary`?